docs/_static/vdr/v2.yaml

```diff
@@ -367,11 +367,27 @@ components:
   schemas:
     CreateDIDOptions:
         type: object
+        description: |
+          Options for the DID creation. If neither `did` nor `user_id` is given, a random UUID is used as user_id.
+          It's invalid to provide both `did` and `user_id` at the same time.
         properties:
-          id:
+          did:
             type: string
-            description: The ID of the DID document. If not given, a random UUID is generated.
-            example: "did:web:example.com:iam:013c6fda-73e8-45ee-9220-48652dba854b"
+            description: |
+              The DID of the DID document. If it's a did:web DID and it contains a user path, it must follow the format `did:web:example.com:iam:1234`.
+              It can be used, for instance, to create a root did:web DID (one without user path).
+            example:
+              - "did:web:example.com"
```

Comment:
How does this work with only one publicURL configured?

Reply:
The easy/most flexible API is: "I want to create this DID: <DID>"

I think we might support multiple domains in future, then this API still applies. I'd add a check that verified the given DID follows the configured `url`.